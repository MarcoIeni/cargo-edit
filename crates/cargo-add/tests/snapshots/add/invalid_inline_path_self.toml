bin.name = "cargo-add"
args = ["add", "."]
status.code = 1
stdout = ""
stderr = """
      Adding cargo-list-test-fixture (local) to dependencies.
Error: Cannot add `cargo-list-test-fixture` as a dependency to itself
"""
fs.sandbox = true

[env.add]
CARGO_IS_TEST="1"
