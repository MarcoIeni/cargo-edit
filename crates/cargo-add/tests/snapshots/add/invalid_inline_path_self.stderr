    Updating '[ROOTURL]/registry' index
      Adding cargo-list-test-fixture (local) to dependencies.
Error: Cannot add `cargo-list-test-fixture` as a dependency to itself
